### DISCUSSION TITLE
Is there any solution to the overflowing when the screen is smaller than the height of the toolbar?

### DISCUSSION THREAD

**Discussion by thewatchers1** [geoman-io/leaflet-geoman#1485] [Questions & Help]
Perhaps collapsing options, a dropdown menu, having the toolbar in a scrollable area? I couldn't find anything relating to this in the docs.

**Comment by Falke-Design** [comment]
You can set the position of specific toolbar containers "blocks" to another place but else this is default Leaflet behavior.

https://www.geoman.io/docs/toolbar
```
map.pm.addControls({  
  positions: {edit: 'topright'}
}); 
```
